src/main/scala/com/scality/clueso/tools/LandingMetadataPopulatorTool.scala

```diff
@@ -0,0 +1,96 @@
+package com.scality.clueso.tools
+
+import java.io.File
+import java.sql.Timestamp
+import java.util.{Date, UUID}
```

Comment:
I'm not a cheatsheet guy, so tend to google a lot when I need to find usage. From StackOverflow to Oracle Java docs, either is fine. Most of the classes I use I know 'by heart' because I've used a lot in the past